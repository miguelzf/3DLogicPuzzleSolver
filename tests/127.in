- - - - - -
- - - - - -
R - P - - -
- - - - - -
- X X X X X
- L - - - -

- - - - - Y
G - - - - -
- - - - G -
- - Y - - -
- - - - - -
- - - - - -

- X - - - -
- X P - - -
- X B - - -
- X R - - -
L X X X X X
- - - - - B
