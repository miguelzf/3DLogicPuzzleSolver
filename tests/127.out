
Solution:
r  r  r  r  r  r  
r  b  b  b  b  b  
R  b  P  p  p  p  
b  b  -  -  -  -  
b  X  X  X  X  X  
b  L  l  l  l  l  
================
b  b  y  y  y  Y  ||  l  X  -  p  b  r  
G  b  y  b  b  b  ||  l  X  P  p  b  r  
g  b  y  b  G  b  ||  l  X  B  b  b  r  
g  b  Y  b  g  b  ||  l  X  R  r  r  r  
g  b  b  b  g  b  ||  L  X  X  X  X  X  
g  g  g  g  g  b  ||  b  b  b  b  b  B  
